boms/integration-tests/src/test/java/com/google/cloud/MaximumLinkageErrorsTest.java

```diff
@@ -46,7 +46,7 @@ public void testForNewLinkageErrors()
     String baselineCoordinates = "com.google.cloud:libraries-bom:" + version;
     Bom baseline = RepositoryUtility.readBom(baselineCoordinates);
 
-    Path bomFile = Paths.get("../cloud-oss-bom/pom.xml");
+    ClassPathEntry bomFile = Paths.get("../cloud-oss-bom/pom.xml");
```

Comment:
I wasn't thinking of BOMs as classpath entries. They're not, are they?

Reply:
They are not. Fixed.